apps/emqx/test/emqx_takeover_SUITE.erl

```diff
@@ -169,3 +306,30 @@ payload(I) ->
             emqx_utils_calendar:now_to_rfc3339(millisecond)
         ])
     ).
+
+%% @doc Filter out the message with matching target payload from the list of messages.
+%%      return '{IsTargetFound, ListOfOtherMessages}'
+%% @end
+-spec filter_payload(List :: [#{payload := binary()}], Payload :: binary()) ->
+    {IsPayloadFound :: boolean(), OtherPayloads :: [#{payload := binary()}]}.
+filter_payload(List, Payload) when is_binary(Payload) ->
+    IsWill = lists:any(
+        fun
+            (#{payload := P}) -> Payload == P;
+            (_) -> false
+        end,
+        List
+    ),
+    Filtered = [
+        Msg
+     || #{payload := P} = Msg <- List,
+        P =/= Payload
+    ],
+    {IsWill, Filtered}.
+
+%% @doc assert emqtt *client* process exits as expected.
+assert_client_exit(Pid, v5) ->
```

Comment:
Tiny nit. Looks like it should be either `assert_client_takenover(Pid, v5)` or `assert_client_exit(Pid, takenover, v5)`, otherwise sounds too broad.